package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "xhtml-purifier",
-	"version": "0.3.1",
+	"version": "0.3.2",
```

Comment:
so, for any open-source project, we should use semantic versioning. See http://semver.org/ if you haven't already